assets/reducers/popup.js

```diff
@@ -9,7 +10,8 @@ const defaultState = {
     point: null,
     isFetchingElevation: false,
     elevation: null,
-    values: []
+    values: [],
+    region: ''
```

Comment:
Okay I think that might have done it, need to do some testing. It looked to me that everything was a "required" prop, so I wasn't sure if that was by design or by coincidence.